subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -273,19 +276,15 @@ static int create_prepare_cb(struct lll_prepare_param *p)
 	struct lll_df_sync_cfg *cfg;
 
 	cfg = lll_df_sync_cfg_latest_get(&lll->df_cfg, NULL);
-#endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
-
-	if (false) {
-#if defined(CONFIG_BT_CTLR_DF_SCAN_CTE_RX)
-	} else if (cfg->is_enabled) {
 
+	if (cfg->is_enabled) {
 		lll_df_conf_cte_rx_enable(cfg->slot_durations, cfg->ant_sw_len, cfg->ant_ids,
 					  chan_idx, CTE_INFO_IN_PAYLOAD);
 		cfg->cte_count = 0;
-#endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
-	} else if (IS_ENABLED(CONFIG_BT_CTLR_DF_SUPPORT)) {
```

Comment:
If I'm correct, then `radio_df_cte_inline_set_enable(false)` should depend on `CONFIG_BT_CTLR_DF_SUPPORT`.
The `CONFIG_BT_CTLR_DF_SCAN_CTE_RX` may be disabled while the `CONFIG_BT_CTLR_DF_SUPPORT` is enabled.
It will allow to synchronize with periodic advertising that includes CTE without sampling of the CTE.

It applies to all places where `radio_df_cte_inline_set_enable(false)` was added.